src/dotnet-scaffolding/dotnet-scaffold-aspnet/Helpers/AspNetDbContextHelper.cs

```diff
@@ -14,21 +14,25 @@ internal class AspNetDbContextHelper
     internal static DbContextProperties SqlServerDefaults = new()
     {
         AddDbMethod = "AddSqlServer",
+        NewDbConnectionString = "Server=(localdb)\\mssqllocaldb;Database={0};Trusted_Connection=True;MultipleActiveResultSets=true"
     };
 
     internal static DbContextProperties SqliteDefaults = new()
     {
         AddDbMethod = "AddSqlite",
+        NewDbConnectionString = "Data Source={0}.db"
     };
 
     internal static DbContextProperties CosmosDefaults = new()
     {
         AddDbMethod = "AddCosmos",
+        NewDbConnectionString = "AccountEndpoint=https://localhost:8081/;AccountKey=C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw=="
```

Comment:
What is this AccountKey (even if it is local only or just a placeholder I'm a little worried about it getting flagged by credscan)?

Reply:
I got this localhost connection string from the official Azure Cosmos DB [docs](https://learn.microsoft.com/en-us/azure/cosmos-db/emulator) regarding running the local emulator.

Reply:
Got it. Well then lets hope credscan already has it allow listed :)